cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-* View high-level information about your virtual machines by
-using the *Virtual Machines* dashboard in the {product-title} web console.
-Access it by navigating to the *Workloads* -> *Virtual Machines* page and
-selecting a virtual machine.
+* Enhancements to workflow for importing a single VMware virtual machine.
```

Comment:
Please make these bullet points all parallel in structure and make sure to use articles where appropriate ("the workflow"). I like full sentences better than fragments. I also think you should group the similar bullet points together, like:

== New and changed features

- The workflow for importing a single VMware virtual machine is enhanced with (can we be more specific here?)
- ...
- ...

You can now:
- Edit virtual machine templates. Previously, only virtual machines could be edited.
- Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
- ...

Thoughts?

Reply:
* The virtual machine creation wizard has been completely redesigned.
It now uses an in-page style and includes a comprehensive review page. 
In addition. you can now:
** Edit virtual machine templates as well as virtual machine configurations.
** Import a single VMware virtual machine with less screen navigation.
** Monitor health of Virtual Machine-backed services, as you would for 
Pod-based services.
** Enable persistent local storage for virtual machine images.
** Add, edit and view virtual CD-ROM devices attached to a virtual machine.
** Add and view network attachment definitions with a graphical editor.

If this is OK, I'll commit. Pinging on Slack.

Reply:
Final version:

* Managing virtual machines is simpler and more efficient due to improvements
in design and workflow. You can now:
** Run the virtual machine wizard with less navigation. The wizard now uses
a comprehensive in-page style and includes a review page for confirming
configuration details before submission.
** Import a single VMware virtual machine with less navigation.
** Edit virtual machine templates as well as virtual machine configurations.
** Monitor health of virtual machine-backed services as you would for
Pod-based services.
** Enable persistent local storage for virtual machine images.
** Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
** Add and view network attachment definitions with a graphical editor.